modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -814,14 +814,16 @@ definition {
 		}
 
 		task ("And given an account, minium site and an order is created") {
+			CommerceJSONAccountsAPI._addCommerceAccount(
+				accountName = "Account Name",
+				accountType = "Business");
+
+			Navigator.openURL();
+
```

Comment:
Hey @andreefarias do we need to go to the home page in order to access the application menu?

Reply:
If we don't, can you address this in your changes?

Reply:
On the test local is not needed, but on CI the failure happen because in this step the test don't see de Application Menu, so I made go back to the home page to see it. What do you think?

Reply:
On the test local is not needed, but on CI the failure happen because in this step the test don't see de Application Menu, so I made go back to the home page to see it. What do you think?

cc: @timpak